mathesar/tests/conftest.py

```diff
@@ -179,3 +179,10 @@ def _create_column(table, column_data):
         column = mathesar_model_column.current_objects.get_or_create(attnum=attnum, table=table)
         return column[0]
     return _create_column
+
+
+@pytest.fixture
+def custom_types_schema(test_db_model, schema, live_server):
+    engine = create_mathesar_engine(test_db_model.name)
+    install.install_mathesar_on_database(engine)
```

Comment:
This will create a `mathesar_types Schema` with all the cast functions. If it is used in multiple successive test cases, there will be an error due to the Schema with the same name already existing. So it has to be dropped after the test case is completed. Refer to `mathesar.tests.conftest.patent_schema` for example on how it is done.